mmv1/templates/terraform/examples/firestore_field_complex_field_name.tf.erb

```diff
@@ -1,7 +1,7 @@
 resource "google_firestore_field" "<%= ctx[:primary_resource_id] %>" {
-  project = "<%= ctx[:test_env_vars]['project_id'] %>"
+  project    = "<%= ctx[:test_env_vars]['project_id'] %>"
   collection = "chatrooms_%{random_suffix}"
-  field = "`*`"
+  field      = "`*`"
 
```

Comment:
This example doesn't actually seem to be referenced in the corresponding YAML file and hasn't been since its creation.

I think it's out-of-scope to fix that now, but I think we should in the future.